src/platforms/monday/actions/graphql/query.ts

```diff
@@ -0,0 +1,25 @@
+import { action } from '@/sdk';
+import { z } from 'zod';
+import client from '../../client';
+
+export default action(
+  'graphql-query',
+  {
+    operation: 'query',
+    resource: 'graphql',
+    mutation: true,
+    schema: z.object({
+      query: z.string(),
+    }),
+    scopes: [],
+  },
+  async ({ auth, input }) => {
+    const result = await client.query(auth, {
+      query: input.query,
+    });
+    return {
+      data: result.data,
+      $native: result.$native,
```

Comment:
is result.data different from $native in this case

Reply:
ah you're right, I need to do `.data.data`